tests/integration/test_requirements_validation.py

```
@@ -0,0 +1,358 @@
+            
+            # Test parsing a workflow file
+            workflow_path = Path(".github/workflows/pr-agent.yml")
+            with open(workflow_path, 'r') as f:
+                workflow = yaml.safe_load(f)
+            
+            assert workflow is not None
+            assert isinstance(workflow, dict)
+            assert 'jobs' in workflow
+        except ImportError:
+            pytest.skip("PyYAML not installed yet")
+        except Exception as e:
```

Ruff (0.14.7)

Do not catch blind exception: `Exception`

(BLE001)